config/errors.yaml

```diff
@@ -7,36 +7,36 @@
 4000:
   :http_status: 404
   :error_code: 4000
-  :description: 'Not found'
+  :description: 'User not found'
 4001:
   :http_status: 400
   :error_code: 4001
   :description: 'Missing Parameter'
-4010:
+4002:
   :http_status: 404
-  :error_code: 4010
+  :error_code: 4002
   :description: 'User can not be deleted or updated due to user not found'
-4011:
+4003:
   :http_status: 401
-  :error_code: 4011
+  :error_code: 4003
```

Comment:
I would change the error description to `User can not be deleted or updated due to unauthorized request`